packages/analytics-plugin-segment/src/browser.js

```diff
@@ -115,26 +117,39 @@ function segmentPlugin(pluginConfig = {}) {
       /* eslint-enable */
     },
     /* Trigger Segment page view http://bit.ly/2LSPFr1 */
-    page: ({ payload }) => {
+    page: ({ payload, config }) => {
       if (typeof analytics === 'undefined') return
 
-      analytics.page(payload.properties)
+      analytics.page(payload.properties, {
+        ...payload.options,
+        integrations: config.integrations,
+      })
     },
     /* Track Segment event http://bit.ly/2WLnYkK */
-    track: ({ payload }) => {
+    track: ({ payload, config }) => {
       if (typeof analytics === 'undefined') return
-      // TODO map options from https://getanalytics.io/api/#analyticstrack to segment https://bit.ly/3lAfjhH
-      analytics.track(payload.event, payload.properties, payload.options)
+
+      analytics.track(payload.event, payload.properties, {
+        ...payload.options,
+        integrations: config.integrations,
```

Comment:
I think the merge needs to be flipped here. This way the individual calls can override the global default

https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/#managing-data-flow-with-the-integrations-object

E.g

```
analytics.track(payload.event, payload.properties, {
   integrations: config.integrations,
   ...payload.options,
})
```

Reply:
Ahh good catch! Will update!